lib/compiler-cranelift/src/translator/translation_utils.rs

```diff
@@ -54,7 +54,7 @@ pub fn type_to_irtype(ty: Type, target_config: TargetFrontendConfig) -> WasmResu
         Type::F32 => Ok(ir::types::F32),
         Type::F64 => Ok(ir::types::F64),
         Type::V128 => Ok(ir::types::I8X16),
-        Type::ExternRef | Type::FuncRef => reference_type(target_config),
```

Comment:
If `ExceptionRef` is passed, should we return an error? (since cranelift doesn't support it)

Reply:
I changed this to 
```
Type::ExternRef | Type::FuncRef => reference_type(target_config),
Type::ExceptionRef => Err(wasm_unsupported!(
            "exnrefs are not supported yet in cranelift"
)),
```
however I think that in practice flow won't reach this call, as the validation that happens before already picks up the presence of `exnref`s, e.g.
```
λ wasmer run php-eh --cranelift
error: Spawn failed
╰─▶ 1: compile error: Validate("exceptions proposal not enabled (at offset 0x9809)")
```